homeassistant/components/izone/climate.py

```diff
@@ -314,12 +317,19 @@ def current_temperature(self) -> Optional[float]:
             return self._controller.temp_supply
         return self._controller.temp_return
 
+    @property
```

Comment:
I'm not so sure about this. It might be better for a template sensor to use the control zone index to get the information from the zone directly, although I guess there's no array of zones to access really.